dadi/lib/handlers/image.js

```diff
@@ -410,22 +410,38 @@ ImageHandler.prototype.convert = function (stream, imageInfo) {
             case 'crop':
               if (options.crop) {
                 var coords = options.crop.split(',').map(coord => parseInt(coord))
-
                 if (coords.length === 2) {
                   coords.push(height - coords[0])
                   coords.push(width - coords[1])
                 }
 
-                sharpImage.extract({
+                var cropDimensions = {
```

Comment:
suggest we use const here, given our gradual migration towards es6

Reply:
I agree!